jsonrpc/eth_endpoint.go

```diff
@@ -172,6 +207,27 @@ func (e *Eth) filterExtra(block *types.Block) error {
 	return nil
 }
 
+// CreateAccessList creates a EIP-2930 type AccessList for the given transaction.
+// Reexec and BlockNrOrHash can be specified to create the accessList on top of a certain state.
+func (e *Eth) CreateAccessList(arg *txnArgs, filter BlockNumberOrHash) (interface{}, error) {
```

Comment:
This function is not really doing anything. The point of `CreateAccessList` should be applying the transaction to the state of provided block in `filter` variable, and trace the access list created by applying that transaction and return it to the caller.

Check how `go-ethereum` does this: https://github.com/ethereum/go-ethereum/blob/c6119247271220ce89e76e1b1b2eaeaaa8fbd9d1/internal/ethapi/api.go#L1463